src/cameras/map/hooks/use-display-case.js

```diff
@@ -0,0 +1,10 @@
+import { useSelector } from 'react-redux'
+
+export const useDisplayCase = () => {
```

Comment:
I don't think this is enough. Picture this scenario:
1. You first visit the page and the cameras get loaded, ergo the `data` field is populated.
2. You want to filter some cameras using the left-side menu, therefore another call is made to the server, thus needing a loading state
3. The call is made, but no loading is displayed, as the `data` field is already populated by the previous call

Let's try to cover this eventuality. Try it on your own first and we'll talk about it if you don't manage to do it.

Reply:
Done